src/cmd/linuxkit/run_qemu.go

```diff
@@ -499,12 +492,12 @@ func buildQemuCmdline(config QemuConfig) (QemuConfig, []string) {
 		}
 	}
 
-	if config.KVM {
-		qemuArgs = append(qemuArgs, "-enable-kvm")
+	if config.Accel != "" {
+		qemuArgs = append(qemuArgs, "-accel", config.Accel)
 		if config.Arch == "aarch64" {
 			qemuArgs = append(qemuArgs, "-machine", "virt,gic_version=host")
 		} else {
-			qemuArgs = append(qemuArgs, "-machine", "q35,accel=kvm:tcg")
+			qemuArgs = append(qemuArgs, "-machine", "q35")
```

Comment:
IIRC `accel=kvm:tcg` meant "try kvm but fallback to tcg" or some such, so should the default for `config.Accel` be `kvm:tcg` rather than just `kvm`?